Makefile

```diff
@@ -20,12 +21,20 @@ HOST_TARGETS := $(INSTALL_WIN_TARGET) $(INSTALL_UNIX_TARGET) $(HOST_APP_TARGET)
 
 all: $(HOST_TARGETS)
 
+install: install-unix
+install-unix: $(HOST_TARGETS)
+	$(INSTALL_UNIX_TARGET) --local $(BROWSER)
+install-win: $(HOST_TARGETS)
+	$(INSTALL_UNIX_TARGET) --local $(BROWSER)
```

Comment:
`install-win` just runs `$(INSTALL_UNIX_TARGET)`...it looks like a failed copy-paste attempt :octocat: ;) s/UNIX/WIN/

Note: `install-win` only makes sense for Windows devs that have `make` available in Powershell (in order to run `make install-win`...). Cygwin or WSL is probably necessary.

Reply:
That said, once using Cygwin, I bet the Unix target would work, too.

Reply:
The script would run on Cygwin, but it would stop at the first check since `uname -s` on Cygwin prints `CYGWIN_FOO` :cry: 
I'm thinking about merging #23 's script with the current Unix installer.